src/components/LmsConfigurations/CornerstoneIntegrationConfigForm.jsx

```diff
@@ -0,0 +1,256 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import {
+  ValidationFormGroup, Input, StatefulButton, Icon,
+} from '@edx/paragon';
+import { snakeCaseFormData } from '../../utils';
+import LmsApiService from '../../data/services/LmsApiService';
+import StatusAlert from '../StatusAlert';
+import SUBMIT_STATES from '../../data/constants/formSubmissions';
+import { handleErrors, validateLmsConfigForm } from './common';
+
+export const REQUIRED_CORNERSTONE_CONFIG_FIELDS = [
+  'cornerstoneBaseUrl',
+];
+
+function configFormReducer(state, action) {
+  switch (action.type) {
+    case 'PENDING': {
+      return { ...state, error: null, submitState: SUBMIT_STATES.PENDING };
+    }
+    case 'ERROR': {
+      return { ...state, error: action.value, submitState: SUBMIT_STATES.ERROR };
+    }
+    case 'INVALID': {
+      return {
+        ...state,
+        submitState: SUBMIT_STATES.ERROR,
+        invalidFields: action.value,
+        error: 'Form could not be submitted as there are fields with invalid values. Please correct them below.',
```

Comment:
Quick check- does this message meet these guidelines? https://openedx.atlassian.net/wiki/spaces/BPL/pages/1966375321/Microcopy+writing+guidelines

Reply:
I think so? The only thing I could think to change is writing "your form" (which sounds bizarre to me since it's not their form) or "The form". I see something about avoiding punctuation for helper text, but I don't count this as "short".

Reply:
Yeah, this definitely qualifies as needing punctuation.